canonical-web-and-design/ubuntu.com
# Adapt the content component for small screens

## Done

- Make the content component responsive to hide the details when on small screens.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10188.demos.haus/advantage?test_backend=true).
- Make your window large and you should see the subscription list and details side by side.
- Resize your window to a mobile size and you should only see the list.
  - Note: the subscription cards and details haven't been updated for the small size yet.


## Issue / Card

Fixes canonical-web-and-design/commercial-squad#92.

## Screenshots

<img width="403" alt="Screen Shot 2021-08-12 at 1 50 38 pm" src="https://user-images.githubusercontent.com/361637/129135751-2c8330eb-492a-48bc-820a-5bcfcdcdbcb3.png">
